docs/current_docs/quickstart/agent/snippets/part2/workspace/java/src/main/java/io/dagger/modules/workspace/Workspace.java

```diff
@@ -52,27 +52,18 @@ public Workspace writeFile(String path, String contents) {
    */
   @Function
   public String listFiles() throws ExecutionException, DaggerQueryException, InterruptedException {
-    return dag()
-      .container()
-      .from("alpine:3")
-      .withDirectory("/src", source)
-      .withWorkdir("/src")
-      .withExec(List.of("tree", "/src"))
-      .stdout();
+    return dag().container().from("alpine:3").withDirectory("/src", source).withWorkdir("/src")
+        .withExec(List.of("tree", "/src")).stdout();
```

Comment:
Not necessarily everywhere, but I think if we can keep the multiline version it would be better.
This might be against the default formatter, but this is only for the docs and I think it's easier to read and to compare with other languages